src/components/select/select.tsx

```diff
@@ -242,6 +242,16 @@ export class Select {
     return this.options.filter(option => option.selected);
   }
 
+  @Method()
+  setSelectedValues(values: string[]) {
+    if (this.multiple) {
```

Comment:
Hey @narendran-kannan Can we make this method common for all the select and not just multi select ?

Reply:
for other select methods, it should accept a single value, not an array of values? Right?
@shravan-balasubramanian

Reply:
Yes!. I have a doubt 
Isn't `document.querySelector('fw-select').value = ['a', 'b', 'c']` working ?
If no, can you make that work instead of another method? thoughts?

Reply:
I can't get this? @shravan-balasubramanian

Reply:
Yes, document.querySelector('fw-select').value = ['1', '2', '3'] is working! @shravan-balasubramanian